Makefile

```diff
@@ -33,6 +33,14 @@ liveness:
 	@echo "-------------------"
 	@ansible-playbook nginx/liveness/liveness.yml -vv
 	@kubectl get pods -n litmus
+
+.PHONY: auxiliaryapp
+auxiliaryapp:
+
+	@echo "------------------------"
+	@echo "Deploying Auxiliary App"
+	@echo "------------------------"
+	@kubectl run mypod --image=nginx -n litmus
```

Comment:
```suggestion
	@kubectl run nginx --image=nginx -n litmus
```

Reply:
nginx in general app I'm deploying another app which is aux app

Reply:
you can set the name as `aux-app`

Reply:
done